Makefile

```diff
@@ -206,3 +214,16 @@ certs/server.pem:
 		-out $@
 
 certs: certs/server.pem
+
+############### Examples/demos
+
+# this example requires some additional Polymer elements
+data/decl-widgets-taxi-demo.ipynb: URTH_COMP_DIR=public/urth_components
+data/decl-widgets-taxi-demo.ipynb:
+	@cp examples/$(@F) data/
```

Comment:
For now, just the one you're dynamically copying. Later, might want to do them all since it's handy to start with an empty server in some cases.